draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2640,24 +2719,42 @@ Internet-Draft                 ACTN POI                    January 2023
       UNI topology discover mechanism defined in [SAP]: this gap has
       been identified in section 4.3;
 
-
-
-Peruzzini et al.        Expires July 11, 2023                 [Page 49]
-
-Internet-Draft                 ACTN POI                    January 2023
-
-
    o  a mechanism applicable to the P-PNC NBI to configure the SR-TE
       paths. Technology-specific augmentations of TE Tunnel model,
       defined in [TE-TUNNEL], are foreseen in section 1 of [TE-TUNNEL]
-      but not yet defined: this gap has been identified in section 5.3.
+      but not yet defined: this gap has been identified in section 5.3;
+
+   o  an attribute to configure the minimum number of active member
+      links required to consider the LAG link as being up is missing
+      from the topology model defined in [CLIENT-TOPO]: this gap has
+      been identified in section 5.2.1;
+
+   o  a mechanism to report client connectivity constraints imposed by
+      some muxponder design: this gap has been identified in appendix
+      Error! Reference source not found..
```

Comment:
same comment as Jeff

Reply:
Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90